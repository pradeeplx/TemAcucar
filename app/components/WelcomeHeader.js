import React from 'react'
import { View, Platform, Dimensions } from 'react-native'
import { fontFactor } from "../helpers"
import Colors from "../Colors"
import Headline from "./Headline"

export default WelcomeHeader = ({ headline, children, style }) => (
  <View style={[{
    backgroundColor: Colors.lightBeige,
    width: Dimensions.get('window').width,
    height: Dimensions.get('window').width * (1780/2000),
    paddingTop: (Platform.OS === 'ios' ? 40 : 30),
    alignItems: 'center',
  }, style]}>
    <Headline style={{
      fontSize: 18 * fontFactor(),
      color: Colors.blue,
      marginBottom: 40,
    }}>
      {headline}
    </Headline>
    { children }
  </View>
)
