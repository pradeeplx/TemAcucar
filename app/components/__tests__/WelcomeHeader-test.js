import 'react-native'
import React from 'react'
import WelcomeHeader from '../WelcomeHeader'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <WelcomeHeader
      style={{top: 0}}
      headline="Foo Bar"
    >
      FooBar
    </WelcomeHeader>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
