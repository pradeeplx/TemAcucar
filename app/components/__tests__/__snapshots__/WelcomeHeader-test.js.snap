exports[`test renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#FAF6F1",
        "height": 667.5,
        "paddingTop": 40,
        "width": 750,
      },
      Object {
        "top": 0,
      },
    ]
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Avenir",
          "fontSize": 18,
        },
        Array [
          Object {
            "color": "#CECED2",
            "fontSize": 24,
            "marginBottom": 20,
            "marginHorizontal": 20,
            "textAlign": "center",
          },
          Object {
            "fontFamily": "Avenir-Medium",
          },
          Object {
            "color": "#44B5B1",
            "fontSize": 27,
            "marginBottom": 40,
          },
        ],
      ]
    }>
    Foo Bar
  </Text>
  FooBar
</View>
`;
